device-types/Dell/PowerEdge-M1000e.yaml

```diff
@@ -4,34 +4,74 @@ model: PowerEdge M1000e
 slug: dell-poweredge-m1000e
 u_height: 10
 is_full_depth: true
+airflow: front-to-rear
+# Maximum weight, until enough device and module models are created. Empty weight 98.1
+weight: 442
+weight_unit: lb
 subdevice_role: parent
+# description: Blade chassis for M630 blades and I/O modules
+comments: |
+  Documentation
+
+  - [Product Support page](https://www.dell.com/support/home/en-us/product-support/product/poweredge-m1000e/overview)
+  - [Dell PowerEdge M1000e Enclosure Owner's Manual](https://www.dell.com/support/manuals/en-us/poweredge-m1000e/m1000eownersmanual/about-your-system)
+interfaces:
+  - name: IOM Slot A1 management backplane
+    type: 1000base-kx
+  - name: IOM Slot A2 management backplane
+    type: 1000base-kx
+  - name: IOM Slot B1 management backplane
+    type: 1000base-kx
+  - name: IOM Slot B2 management backplane
+    type: 1000base-kx
+  - name: IOM Slot C1 management backplane
+    type: 1000base-kx
+  - name: IOM Slot C2 management backplane
+    type: 1000base-kx
+  - name: Chassis CMC
+    type: other
+    mgmt_only: true
 power-ports:
-  - name: PSU1
+  - name: PS-1
     type: iec-60320-c20
-  - name: PSU2
+  - name: PS-2
     type: iec-60320-c20
-  - name: PSU3
+  - name: PS-3
     type: iec-60320-c20
-  - name: PSU4
+  - name: PS-4
     type: iec-60320-c20
-  - name: PSU5
+  - name: PS-5
     type: iec-60320-c20
-  - name: PSU6
+  - name: PS-6
     type: iec-60320-c20
```

Comment:
Please correct me if I am wrong, but it looks like the PSUs are modular. If that is true, these should be defined as `module-bays` and the associated PSUs should be created in the module-types folder